scripts/autogenHelpers/DocJSONBuilder.js

```diff
@@ -26,24 +26,23 @@ const IGNORE_FILES = [
 
 const IGNORE_METHODS = ['setNativeProps'];
 
+const fileExtensionsRegex = /.(js|tsx|(?<!d.)ts)$/;
+
 class DocJSONBuilder {
   constructor(styledLayers) {
     this._styledLayers = {};
 
     for (const styleLayer of styledLayers) {
       const ComponentName = pascelCase(styleLayer.name);
       this._styledLayers[
-        ComponentName +
-          (['Light', 'Atmosphere', 'Terrain'].includes(ComponentName)
-            ? ''
-            : 'Layer')
+        ComponentName + (ComponentName === 'Light' ? '' : 'Layer')
```

Comment:
I think this is a merge error, 
```suggestion
        ComponentName +
          (['Light', 'Atmosphere', 'Terrain'].includes(ComponentName)
            ? ''
            : 'Layer')
```